src/launcher_config.js

```diff
@@ -45,7 +45,8 @@ const defaultSetup = {
 		'engine': undefined,
 		'map_options': undefined,
 		'mod_options': undefined,
-		'game_options': undefined
+		'game_options': undefined,
+		'springsettings': {}
```

Comment:
So as I understand it, the config priority order is:
1. config.json's config-based springsettings (this), distributed by the app and overridable by devs
2. User's springsettings.cfg
3. springsettings.json distributed by the app

Is it desirable for 1 to override 2? This also means that LuaMenu's/game's config would continuously overriden by launcher ("default") settings, which might not be what users want. 

In any case, I think we may want to document this somewhere :) A simple in-code comment would be a good start imo!

Reply:
Added description of the option.

The purpose of this option is to exactly override whatever was previously specified in the option. it allows for different config selections to force particular, different between config settings. The purpose is to support overriding `RapidTagResolutionOrder` option.